osu.Game/Screens/Edit/Timing/ControlPointTable.cs

```diff
@@ -44,6 +44,7 @@ public IEnumerable<ControlPointGroup> ControlGroups
                 {
                     BackgroundFlow.Add(new RowBackground(group)
                     {
+                        Selected = ReferenceEquals(group, selectedGroup?.Value),
```

Comment:
The null check on `selectedGroup` here is causing code quality fail. I'd hope this isn't actually needed, but if it is then moving this to an `updateDisplay` or `Schedule`ing is a better option.